<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-21T22:53:54 IST" name="Default suite" finished-at="2022-03-21T22:54:53 IST" duration-ms="59233">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@316bcf94]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2022-03-21T22:53:54 IST" name="Default test" finished-at="2022-03-21T22:54:53 IST" duration-ms="59233">
      <class name="runner.TestRunner">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:runner.TestRunner@316bcf94]" started-at="2022-03-21T22:53:54 IST" name="setUpClass" finished-at="2022-03-21T22:53:56 IST" duration-ms="1708" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@316bcf94]" started-at="2022-03-21T22:53:56 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-03-21T22:54:51 IST" duration-ms="55005" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Arcs text link test of Api demos apk"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Graphics testing of api demo apk]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at stepsdefinitions.GraphicsTest.open_arc_screen(GraphicsTest.java:63)
at âœ½.open arc screen(file:///C:/Users/rajibmahato/git/AppiumTest/AppiumExitTest/./src/test/java/features/Graphics.feature:8)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runner.TestRunner@316bcf94]" started-at="2022-03-21T22:54:51 IST" name="tearDownClass" finished-at="2022-03-21T22:54:53 IST" duration-ms="2347" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
